### PR TITLE
Solution

### PR DESCRIPTION
**Pull request by HStakhiv**

### CODE REVIEW

**Review by FranchukS — APPROVED** [review]
GJ!

File: app/main.py [hunk 1]
```diff
@@ -2,4 +2,67 @@ class Cargo:
     def __init__(self, weight):
```
Comment (Nattalli) [inline app/main.py:2]: add type annotation

**Review by Nattalli — CHANGES_REQUESTED** [review]

File: app/main.py (hunk 1 above)
Comment (HStakhiv) [reply app/main.py]: Done

**Review by Nattalli — APPROVED** [review]
Well done!

**Pull request closed without merging**